AEPMessaging/Sources/Messaging.swift

```diff
@@ -428,19 +531,26 @@ public class Messaging: NSObject, Extension {
                 // update rules in in-app engine
                 rulesEngine.launchRulesEngine.replaceRules(with: allInAppRules)
 
-            case .feed:
-                Log.trace(label: MessagingConstants.LOG_TAG, "Updating feed definitions for surfaces \(newRules.compactMap { $0.key.uri }).")
+            case .feed, .contentCard:
+                Log.trace(label: MessagingConstants.LOG_TAG, "Updating content card definitions for surfaces \(newRules.compactMap { $0.key.uri }).")
 
-                // replace rules for each feed surface we got back
-                feedRulesBySurface.merge(newRules) { _, new in new }
+                // replace rules for each content card surface we got back
+                contentCardRulesBySurface.merge(newRules) { _, new in new }
 
                 // remove any surfaces that were requested but had no in-app content returned
                 for surface in surfacesToRemove {
-                    feedRulesBySurface.removeValue(forKey: surface)
+                    contentCardRulesBySurface.removeValue(forKey: surface)
                 }
 
-                // update rules in feed rules engine
-                feedRulesEngine.launchRulesEngine.replaceRules(with: feedRulesBySurface.flatMap { $0.value })
+                // update rules in content card rules engine
+                contentCardRulesEngine.launchRulesEngine.replaceRules(with: contentCardRulesBySurface.flatMap { $0.value })
+
+                // process a generic event to see if there are any content cards with no client-side qualification
+                let genericEvent = Event(name: "Seed content cards", type: EventType.edge, source: EventSource.requestContent, data: nil)
```

Comment:
nit: We change the event type to Messaging in Android. Maybe we want to do the same here?

Reply:
this event doesn't get dispatched - it's really just here to get cards from the rules engine that the user has previously qualified for :) 

i'll change for consistency though.